apps/web/components/booking/pages/BookingPage.tsx

```diff
@@ -584,23 +585,27 @@ const BookingPage = ({
                     <div className="text-bookinghighlight flex items-start text-sm">
                       <FiUser
                         className={`ml-[2px] mt-[2px] inline-block h-4 w-4 ltr:mr-[10px] rtl:ml-[10px] ${
-                          booking && booking.attendees.length / eventType.seatsPerTimeSlot >= 0.5
+                          currentSlotBooking &&
```

Comment:
Why are we introducing `currentSlotBooking`? Wouldn't we have the attendee information already from the `booking` prop?